mpu6050_user/get_temp_mpu6050.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+display_help() {
+  echo
+  echo "Usage: $0"
+  echo
+  echo "    --C, get temperature in Celsius "
+  echo "    --F, get temperature in Fahrenheit "
+  echo "    -h, --help - print these options"
+  echo
+  echo
+}
+
+if [ "$1" = "--help" ] || [ "$1" = "--h" ] ||\
+	[[ "$1" != "--C" && "$1" != "--F" ]] ; then
+	display_help
+	exit 1
+fi
+
+i2cdetect -y 0
+i2cset -y 0 0x68 0x6b 0x80
+i2cset -y 0 0x68 0x6b 0
+
+while true; do
+
+th16=$(i2cget -y 0 0x68 0x41)
+
+th=$((10#$((th16))))
+
+tl16=$(i2cget -y 0 0x68 0x42)
+
+tl=$((10#$((tl16))))
+
+if [ th > 127 ]; then
+	th=$(($th-256))
+	tl=$(($tl-256))
+fi
```

Comment:
Please check the correctness of this conversation.
if [ th > 127 ]; then
	th=$(($th-256))
	tl=$(($tl-256))
fi